plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergDynamicPartitionPruningTest.java

```diff
@@ -66,9 +60,27 @@ protected void createPartitionedTable(String tableName, List<String> columns, Li
         getQueryRunner().execute(sql);
     }
 
+    @Override
+    protected void createPartitionedAndBucketedTable()
+    {
+        // Use decimal type because Iceberg doesn't allow float type on bucket columns
```

Comment:
Maybe we can change the parent class to bucket on decimal and thus avoid overriding `createPartitionedAndBucketedTable`

@sopel39 wdyt?

Reply:
bigint?